kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/utils/_String.kt

```diff
@@ -0,0 +1,75 @@
+@file:JvmMultifileClass
```

Comment:
Almost none of the changes to this file were necessary, I'd say. The boolean parsing was fine and, as I mentioned on Discord, is going to be replaced when we eventually get around to i18n support.